sdk/servicebus/microsoft-azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/SharedAccessAuthorizationRule.java

```diff
@@ -55,7 +55,7 @@ public String getKeyName() {
     @Override
     public void setKeyName(String keyName) {
         if (keyName == null || keyName.isEmpty()) {
```

Comment:
I would replace this with Replace with Objects.requireNotNull and then another check for isEmpty.